app/models/reservation.rb

```diff
@@ -1,4 +1,9 @@
 class Reservation < ApplicationRecord
   belongs_to :user
   belongs_to :court
+  scope :school, -> { where(school_reservation: true) }
```

Comment:
slabe ten nazewnictwo, bo w pierwszej chwili sugeruje ze `school` i `people` to sa relacje